conduct.md

```diff
@@ -23,57 +23,68 @@ permission or without crediting them) such code will negatively impact
-    * Prevent peers from voicing their opinions and questions.
-    * Violate any part of this Code of Conduct.
-    * Violate the Allegheny honor code in regards to all submitted work.
-    * :heavy_exclamation_mark: TODO: add any additional items in the U category.
+- Acceptable Conduct (A):
+
+  - Members shall
+    - Show respect for all peers as learners and team members.
+    - Allow peers to voice their opinions and questions.
+    - Read, understand, and follow all parts of this code of Conduct.
+    - Follow the honor code on all submitted work.
+    - :heavy_exclamation_mark: TODO: add any additional items in the A category.
+
+- Unacceptable Conduct (U):
+  - Members shall not
+    - Disrespect peers because of their race or gender.
+    - Prevent peers from voicing their opinions and questions.
+    - Violate any part of this Code of Conduct.
+    - Violate the Allegheny honor code in regards to all submitted work.
+    - :heavy_exclamation_mark: TODO: add any additional items in the U category.
 
 #### Attendance
 
-* Acceptable Conduct (A):
-  * Members shall
-    * :heavy_exclamation_mark: TODO: add items in the A category for attendance.
+- Acceptable Conduct (A):
+
+  - Members shall
+    - :heavy_exclamation_mark: TODO: add items in the A category for attendance.
 
-* Unacceptable Conduct (U):
-  * Members shall not
-    * :heavy_exclamation_mark: TODO: add items in the U category for attendance.
+- Unacceptable Conduct (U):
+  - Members shall not
+    - :heavy_exclamation_mark: TODO: add items in the U category for attendance.
 
 #### Deadlines
 
-* Acceptable Conduct (A):
-  * Members shall
-    * :heavy_exclamation_mark: TODO: add items in the A category for deadlines.
+- Acceptable Conduct (A):
 
-* Unacceptable Conduct (U):
-  * Members shall not
-    * :heavy_exclamation_mark: TODO: add items in the U category for deadlines.
+  - Members shall
+    - Turn assignments in on time.
+    - Complete their work in a timely manner that will not negatively affect other group members.
+
+- Unacceptable Conduct (U):
+  - Members shall not
+    - Be dishonest about late assignments.
+    - Bargain for more time to complete assignments.
 
 #### Communication
 
 This section defines what communication is acceptable and unacceptable. It is vital that team members not only communicate clearly, but communicate using the right platforms and methods, such as GitHub issues tracker or slack. For example, something may be clearly communicated in Slack, but may still receive an 'unacceptable' because Slack was the inappropriate platform to use.
 
-* Acceptable Conduct (A):
-  * Members shall
-    * Use appropriate softwares for electronic communication.
-    * Always participate in the discussion.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for communication.
+- Acceptable Conduct (A):
+
+  - Members shall
+    - Use appropriate softwares for electronic communication.
```

Comment:
I think it's good to be specific here. What method(s) of communication should the team use? Slack, GitHub Issue Trackers, email, all of them?